packages/react-search-ui-views/package.json

```diff
@@ -30,7 +31,7 @@
     "watch-js": "./bin/watch-js",
     "build-js": "./bin/build-js",
     "build": "run-s clean build-css build-js",
-    "watch": "npm run clean && run-p watch-css watch-js",
+    "watch": "npm run build && run-p watch-css watch-js",
```

Comment:
do you think we need a `run build` step before the `watch`? watch-js runs the same command as the build step, just includes a watch flag?

Reply:
Exactly! That's what I tried at first but it didn't work for me. TBH I don't really follow what happens under the hood here, and just want to make this command work even at a cost of little inefficiency. 🤷